Jenkinsfile

```diff
@@ -0,0 +1,154 @@
+import groovy.json.JsonOutput
+
+/** Tox environment */
+def environment = 'tests'
```

Comment:
Fixed, I think (at least, tests run locally and on Travis).  They didn't get past setup.py on Jenkins staging, but hopefully that's a separate, fixable issue.